<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#background{
background-color: qlineargradient(spread:pad, x1:0.323751, y1:0.188, x2:0.727, y2:0.977409, stop:0 rgba(192, 192, 192, 255), stop:1 rgba(255, 255, 255, 255));
}

QLabel#mainTitle{
font: 24pt &quot;MS Reference Sans Serif&quot;;
}

QWidget#Container{
background-color: #ffffff;
border-radius: 10px;
}

QLabel#ContainerTitle{
font: 16pt &quot;MS Reference Sans Serif&quot;;
}

QObject#answerBox{
border-radius : 5px;
font: 16pt &quot;MS Reference Sans Serif&quot;;
background-color:rgba(225, 225, 225,90);
padding-left:20px;
}

QPushButton{
background-color: #ffffff;
}

</string>
    </property>
    <widget class="QLabel" name="mainTitle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Hiragana and Katakana Trainer</string>
     </property>
    </widget>
    <widget class="QWidget" name="Container" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>1101</width>
       <height>501</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="answerBox">
      <property name="geometry">
       <rect>
        <x>395</x>
        <y>320</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="question">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>381</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>100</pointsize>
       </font>
      </property>
      <property name="text">
       <string>„ÅÇ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>492</y>
        <width>1140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>53</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>20</y>
        <width>211</width>
        <height>161</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>191</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="statTitle">
          <property name="font">
           <font>
            <family>MS Reference Sans Serif</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
          </property>
          <property name="text">
           <string>Stats</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mode">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Current Mode : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalAttempts">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Total Attempts : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalUnattempts">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Questions Left : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="score">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Current Score : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>190</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>quit.png</normaloff>quit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="soundButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>180</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <activeoff>unmute.png</activeoff>
        <activeon>mute.png</activeon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Ctrl+M</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="submit">
      <property name="geometry">
       <rect>
        <x>682</x>
        <y>325</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>enter.png</normaloff>enter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>99</width>
        <height>36</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
